modules/rostests/apitests/powrprof/power.c

```diff
@@ -60,7 +60,7 @@ void test_CallNtPowerInformation(void)
    /* LastSleepTime tests */
    retval = CallNtPowerInformation(LastSleepTime, 0, 0, 0, 0);
    ok(retval == STATUS_SUCCESS, "function result wrong expected STATUS_SUCCESS but got %d\n", (UINT)retval);
-   retval = CallNtPowerInformation(LastSleepTime, &atime, sizeof(sizeof(ULONGLONG)), 0, 0);
+   retval = CallNtPowerInformation(LastSleepTime, &atime, sizeof(atime), 0, 0);
```

Comment:
I agree that `sizeof(variable)` is generally preferable to `sizeof(TYPE)` but it's still a style choice, and the #1 rule of style is to be consistent, so please use `sizeof(ULONGLONG)`. Then this becomes a trivial fix.

Reply:
Reverted.